xdp/src/netlink.rs

```diff
@@ -88,8 +77,10 @@ impl NetlinkSocket {
         Ok(())
     }
 
-    fn recv(&self) -> Result<Vec<NetlinkMessage>, io::Error> {
-        let mut buf = [0u8; 4096];
+    pub(crate) fn recv(&self) -> Result<Vec<NetlinkMessage>, io::Error> {
+        // create large buffer for when we poll the socket
```

Comment:
this comment is still off (nothing to do with polling). I think somewhere you
found that 8k is the right magic number? Can you link that place in a comment here?